app/src/main/cpp/skyline/gpu/texture/texture.h

```diff
@@ -80,7 +80,7 @@ namespace skyline::gpu {
              * @return The size of the texture in bytes
              */
             constexpr size_t GetSize(u32 width, u32 height, u32 depth = 1) const {
-                return (((width / blockWidth) * (height / blockHeight)) * bpb) * depth;
+                return ((((width + blockWidth - 1) / blockWidth) * ((height + blockHeight - 1) / blockHeight)) * bpb) * depth;
```

Comment:
I'd say separating this into `util::DivideCeil` would be more neat, place it after `IsWordAligned` in `common/utils.h`. It can just be `util::DivideCeil(width, blockWidth) * util::DivideCeil(height, blockHeight) * bpb * depth` at that point.